.gitpod.yml

```diff
@@ -19,6 +19,11 @@ tasks:
     command: >
       export IHP_BASEURL=`gp url 8000`;
       export IHP_IDE_BASEURL=`gp url 8001`;
+      # Compile CSS
```

Comment:
@mpscholten maybe you'd know that. `npm` is available only after we run `./start`. I tried to `nix-shell --run 'make -B .envrc'` before but didn't seem to install.

Reply:
Not entirely sure, but you could try `nix-shell --run 'npm install'` maybe :)